app/src/main/java/com/naman14/timberx/util/Utils.kt

```diff
@@ -33,96 +26,37 @@ import java.util.Collections
 
 object Utils {
 
-    val MUSIC_ONLY_SELECTION = (MediaStore.Audio.AudioColumns.IS_MUSIC + "=1" +
+    const val MUSIC_ONLY_SELECTION = (MediaStore.Audio.AudioColumns.IS_MUSIC + "=1" +
             " AND " + MediaStore.Audio.AudioColumns.TITLE + " != ''")
+    const val EMPTY_ALBUM_ART_URI = "android.resource://com.naman14.timberx/drawable/icon"
 
     fun getAlbumArtUri(albumId: Long): Uri {
         return ContentUris.withAppendedId(Uri.parse("content://media/external/audio/albumart"), albumId)
     }
 
     fun makeShortTimeString(context: Context, _secs: Long): String {
```

Comment:
Why `_`?

Reply:
@Zhuinden for `EMPTY_ALBUM_ART_URI`? Because it's a constant

Reply:
No, for `_secs` :smile:

Reply:
@Zhuinden ohhh, no idea, @naman14 had that

Reply:
Only because `seconds` undearneath was called `secs` too i think